x/nft/keeper/keeper_test.go

```diff
@@ -28,24 +33,39 @@ const (
 type TestSuite struct {
 	suite.Suite
 
-	app         *simapp.SimApp
 	ctx         sdk.Context
 	addrs       []sdk.AccAddress
 	queryClient nft.QueryClient
+	nftKeeper   keeper.Keeper
 }
 
 func (s *TestSuite) SetupTest() {
-	app := simapp.Setup(s.T(), false)
+	var (
+		interfaceRegistry codectypes.InterfaceRegistry
+		bankKeeper        bankkeeper.Keeper
+		stakingKeeper     *stakingkeeper.Keeper
+		nftKeeper         keeper.Keeper
+	)
+
+	app, err := simtestutil.Setup(
+		testutil.AppConfig,
```

Comment:
Somehow when accessing `testutil.AppConfig` from here, it fails here https://github.com/cosmos/cosmos-sdk/blob/main/core/appconfig/config.go#L24-L27, so this does not work yet.

The rest is ok to review already.

Reply:
Thank you very much! That fixes it!